tests/test_wave_03.py

```diff
@@ -54,13 +54,15 @@ def test_friends_unique_movies_not_duplicated():
 
     # Assert
     assert len(friends_unique_movies) == 3
+    assert INTRIGUE_3 in friends_unique_movies
```

Comment:
Great start! It would be a good idea to check to see if the other two movies in the list are what you expect as well.